### PR TITLE
H4ck's model and textures update

### LINKED ISSUE #1845: [FEATURE] Update H4ck's model and textures
## 🚀 Feature
Update the model and textures for H4ck.

https://drive.google.com/drive/u/1/folders/1v2tyX-RHox3pv7ddYro6HoBSuh7otDlP

### PR DESCRIPTION
**Pull request by saugon**
Closes #1845 

## Motivation
There is a new version of the model and textures for H4ck.

## Summary of changes
- Added new model.
- Added new textures.
- Updated the material.
- Updated in-game and ui prefabs (and theirs references).
- Animation controller minor fix.

## How has this been tested?
- H4ck's textures are the new ones (check reference).
- The model is working properly in the UI.
- The model is working properly in game. 

## Reference

https://github.com/lambdaclass/champions_of_mirra/assets/145453767/f767f21e-983b-4b8c-8fd8-d86a8e8fe2ca

## Checklist
- [x] I have tested the changes locally.
- [x] I have tested the whole game after applying the changes, not only the affected areas.
- [ ] I self-reviewed the changes on GitHub, line by line.
- [x] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [x] Tested in Android.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Comment by tkz00** [comment]
I've tested it on my android device, it works and looks good!

**Comment by tkz00** [comment]
One issue that was found is H4ck's slingshot is floating detached from it's hand during it's defeat animation in the end game splash screen.

https://github.com/lambdaclass/champions_of_mirra/assets/67983386/9b2c0b67-087e-4d8c-a841-1d8d913da30c

**Review by tkz00 — APPROVED** [review]
Great! Looks good on my android device!

File: client/Assets/Animations/Hack/H4ck Ulti v3.fbx
Comment (AngieDutra) [inline client/Assets/Animations/Hack/H4ck Ulti v3.fbx:1]: I was about to create an issue on this, awesome catch!

File: client/Assets/Prefabs/VisualEffects/H4ck_Denial_AOE.prefab
Comment (AngieDutra) [inline client/Assets/Prefabs/VisualEffects/H4ck_Denial_AOE.prefab:1]: Just to check: this is an intended modification or it part of the unwanted modified files?

File: client/Assets/VFX-Tools/PIngle_VFX/DamageZone/M_SandFlow.mat
Comment (AngieDutra) [inline client/Assets/VFX-Tools/PIngle_VFX/DamageZone/M_SandFlow.mat:1]: I think this was not intended to be pushed

File: client/ProjectSettings/GvhProjectSettings.xml
Comment (AngieDutra) [inline client/ProjectSettings/GvhProjectSettings.xml:1]: Was this intended to be pushed?

File: client/Assets/Prefabs/VisualEffects/H4ck_Denial_AOE.prefab
Comment (saugon) [reply client/Assets/Prefabs/VisualEffects/H4ck_Denial_AOE.prefab:1]: Fixed!

File: client/Assets/VFX-Tools/PIngle_VFX/DamageZone/M_SandFlow.mat
Comment (saugon) [reply client/Assets/VFX-Tools/PIngle_VFX/DamageZone/M_SandFlow.mat:1]: Fixed!

File: client/ProjectSettings/GvhProjectSettings.xml
Comment (saugon) [reply client/ProjectSettings/GvhProjectSettings.xml:1]: Fixed!

**Review by tkz00 — APPROVED** [review]
Great! Looks good on my android!

**Review by alfopisano — APPROVED** [review]
Nice!! Looks great on my iOS device

**Pull request merged**